app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts.length)
+  end
+
+  def five_recent_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] [OPTIONAL] You can use one of the Rails Active Callbacks like `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method private